doc/test-framework/scorecard.md

```diff
@@ -110,54 +94,81 @@ The hierarchy of config methods for the global options that are also configurabl
 
 The config file support is provided by the `viper` package. For more info on how viper configuration works, see [`viper`'s README][viper].
 
-## What Each Internal Plugin Test Does
+**NOTE:** The config file can be in any of the `json`, `yaml`, or `toml` formats as long as the file has the correct extension. As the config file may be extended to allow configuration
+of all `operator-sdk` subcommands in the future, the scorecard's configuration must be under a `scorecard` subsection. 
 
-There are 8 internal tests across 2 internal plugins that the scorecard can run. If multiple CRs are specified for a plugin, the test environment is fully cleaned up after each CR so each CR gets a clean testing environment.
+### Command Args
 
-### Basic Operator
+While most configuration is done via a config file, there are a few important args that can be used as follows. 
 
-#### Spec Block Exists
+| Flag        | Type   | Description   | 
+| --------    | -------- | -------- |
+| `--config`  | string | Path to config file (default `<project_dir>/.osdk-scorecard`; file type and extension can be any of `.yaml`, `.json`, or `.toml`). If a config file is not provided and a config file is not found at the default location, the scorecard will exit with an error. |
+| `--output`, `-o`  | string | Output format. Valid options are: `text` and `json`. The default format is `text`, which is designed to be a simpler human readable format. The `json` format uses the JSON schema output format used for plugins defined later in this document. |
+| `--kubeconfig`, `-o`  | string |  path to kubeconfig. It sets the kubeconfig internally for internal plugins and sets the `KUBECONFIG` env var to the provided value for external plugins. If an external plugin specifically sets the `KUBECONFIG` env var, the kubeconfig from the specified env var will be used for that plugin instead. |
 
-This test checks the Custom Resource(s) created in the cluster to make sure that all CRs have a spec block. This test
-has a maximum score of 1.
+### Config File Options
 
-#### Status Block Exists
+| Option        | Type   | Description   | 
+| --------    | -------- | -------- |
+| `output` | string | equivalent of the `--output` flag. If this option is defined by both the config file and the flag, the flag's value takes priority |
+| `kubeconfig` | string | equivalent of the `--kubeconfig` flag. If this option is defined by both the config file and the flag, the flag's value takes priority |
+| `plugins` | array | this is an array of [Plugins](#plugins).|
 
-This test checks the Custom Resource(s) created in the cluster to make sure that all CRs have a status block. This
-test has a maximum score of 1.
+### Plugins
 
-#### Writing Into CRs Has An Effect
+A plugin object is used to configure plugins. The possible metadatas for the plugin object are `basic`, `olm`, or `external`.
```

Comment:
maybe reword:  "....The possible values for the plugin object are ...."

Reply:
Tks 👍